pkgs/games/umoria/default.nix

```diff
@@ -21,6 +22,15 @@ gcc9Stdenv.mkDerivation rec {
     sha256 = "sha256-1j4QkE33UcTzM06qAjk1/PyK5uNA7E/kyDe3bZcFKUM=";
   };
 
+  patches = [
```

Comment:
https://github.com/dungeons-of-moria/umoria/commit/f294e5880cd21d25c11eee820d629f4ff504ad10 this commit will restore darwin support
Should apply cleanly

Reply:
Applied, and a few more hacks on top to get `pkgsLLVM.umoria` build on `linux`.